File: week11/week11-1.py
def isprime(n):
  for i in range(2,n):
    if n%i==0:return 0
  return 1

bound=500000
ans=0
for i in range(2,bound):
  if isprime(i):
    print(i,end=' ')
    ans+=1
print()
print("質數有",ans,"個")

File: week11/week11-1.cpp
#include <stdio.h>
int isprime(int n){
    for(int i=2;i<n;i++){
        if(n%i==0) return 0;
    }
    return 1;
}
int main()
{
    int bound = 10000,ans=0;
    for (int i=2;i<bound;i++){
        if(isprime(i)){ printf("%d ",i);
            ans++;
        }
    }
    printf("´Ó¾ð¦³:%d­Ó\n",ans);
}
